model/aws/sdk-spans.yml

```diff
@@ -297,3 +297,57 @@ groups:
         requirement_level: recommended
       - ref: aws.s3.part_number
         requirement_level: recommended
+  
+  - id: span.aws.sqs.client
+    extends: span.aws.client
```

Comment:
JFYI: there is an active issue against this span definition - https://github.com/open-telemetry/semantic-conventions/issues/1545, from semconv perspective we'd like to fix it at some point which may cause breaking changes if you use `rpc.*` attributes in AWS SDK instrumentations.

would appreciate your input on this.

Reply:
So for this, why don't we just add `aws-api` to the list of known values here: https://github.com/open-telemetry/semantic-conventions/blob/520702f1557743fa05588999757106007fb21ba3/model/rpc/registry.yaml#L216-L240 and maybe update this md file https://github.com/open-telemetry/semantic-conventions/blob/v1.28.0/docs/cloud-providers/aws-sdk.md to make it less confusing? 

We should probably keep the rpc.system/rpc.operation for AWS because it's heavily used my AWS SDKs in multiple languages and instrumentation agents looks for these values. Changing the semcov will break all of these which wouldn't be ideal.

Reply:
AWS SDK works on top of HTTP, correct? It's not an RPC protocol or middleware, but an SDK on top of REST API, so it's not an RPC system - PTAL at the https://github.com/open-telemetry/semantic-conventions/pull/1622#pullrequestreview-2503504000

Reply:
fixed